_opensearch/query-dsl/index.md

```diff
@@ -120,3 +120,38 @@ With query DSL, however, you can include an HTTP request body to look for result
 }
 ```
 The OpenSearch query DSL comes in three varieties: term-level queries, full-text queries, and boolean queries. You can even perform more complicated searches by using different elements from each variety to find whatever data you need.
+
+## A note on special characters in field values 
+
+Due to word boundaries associated with Unicode special characters, the Unicode standard analyzer cannot index [text field type](https://opensearch.org/docs/2.2/opensearch/supported-field-types/text/) values as whole values when they contain these special characters. As a result, a field value of this type that includes a special character is parsed by the standard analyzer as multiple values separated by the special character, effectively tokenizing the different elements either side of it. This can lead to unintentional requests and search query outcomes, including those that can have an impact on security. 
```

Comment:
> This can lead to unintentional requests and search query outcomes, including those that can have an impact on security.

This feels really vague, what amount something more specific? `If authoring a document filtering query [killer example doc link]() avoiding special characters in the filter conditions of text fields, otherwise the filtering logic could filter unexpected documents.`

Reply:
I'm at a loss for an example of the sort you mention above (probably why it came across as vague). Would this be any better?
"This can lead to unintentional filtering of documents and potentially compromise security safeguards that control access."